_locales/en/messages.json

```diff
@@ -1369,7 +1369,7 @@
 		"message": "Sign In"
 	},
 	"hub_side_navigation_log_out": {
-		"message": "Sign Out"
+		 "message": "Sign Out"
```

Comment:
I think you typed a space here by accident 😶

Reply:
Whaaa! How, Zarembsky!

Reply:
Fixed